images/metering-ansible-operator/.travis.yml

```diff
@@ -0,0 +1,7 @@
+sudo: required
```

Comment:
Delete this file, we aren't using travis, and I'm not sure if we'll be using any of the molecule test stuff yet.